content/guides/3.auth/2.access-control.md

```diff
@@ -92,6 +92,12 @@ An admin can set the user status. Only the active state is able to authenticate,
 
 A policy can also have a specific allowlist of IP addresses, IP ranges, and CIDR blocks which allow access. Leave this empty to allow all IP addresses.
 
+::callout{icon="material-symbols:info-outline"}
+
+**Specify both IPv4 and IPv6**<br/>
+In order to avoid being unable to access Directus, be sure to specify both IP address variants.
```

Comment:
```suggestion

IPv4 and IPv6 are different ways devices are identified on the internet.

Specifying both the IPv4 and IPv6 addresses ensures uninterrupted access to your Directus project by accommodating various network configurations and preventing potential connectivity issues arising from disabling either protocol.
```

Reply:
@phazonoverload thank you for the feedback! What do you think of this?

Reply:
LGTM